### PR TITLE
Mouse input

### PR DESCRIPTION
**Pull request by dariandzirko**
Now we can use a mouse although I am sure a track pad will not be a good time.

### CODE REVIEW

File: src/constants.rs [hunk 1]
```diff
@@ -3,7 +3,7 @@ pub const BASE_SPEED: f32 = 200.;
 pub const TIME_STEP: f32 = 1. / 60.;
 pub const BLASTER_SHOT_HEAT_ADDITION: f32 = 5.;
 pub const BLASTER_POWER_SHOT_THRESHOLD: f32 = 90.;
-pub const MAX_BLASTER_HEAT: f32 = 100.;
+pub const MAX_BLASTER_HEAT: f32 = 1000.;
```
Comment (ssnover) [inline src/constants.rs]: Wanna take this out or try to make it configurable somehow?

File: src/player.rs [hunk 2]
```diff
@@ -86,7 +103,9 @@ fn player_control_system(
             velocity.x = 0.;
         }
 
-        weapon_data.firing = keys.pressed(KeyCode::RShift);
+        //weapon_data.firing = keys.pressed(KeyCode::RShift);
```
Comment (ssnover) [inline src/player.rs]: Should delete.

File: src/player.rs [hunk 3]
```diff
@@ -45,13 +45,30 @@ fn player_spawn_system(
...
     buttons: Res<Input<GamepadButton>>,
+    mouse_buttons: Res<Input<MouseButton>>,
+    mut cursor_evr: EventReader<CursorMoved>,
+    win_size: Res<WindowSize>,
+    windows: Res<Windows>,
 ) {
-    let (mut velocity, mut weapon_data) = query.get_single_mut().unwrap();
+    let (mut velocity, mut weapon_data, player_tf) = query.get_single_mut().unwrap();
+
+    // for ev in cursor_evr.iter() {
```
Comment (ssnover) [inline src/player.rs]: Should delete

**Review by ssnover — CHANGES_REQUESTED** [review]
Constant is still wrong

**Comment by dariandzirko** [comment]
> Constant is still wrong

This is dumb overheating while prototyping is annoying

**Review by ssnover — APPROVED** [review]

**Pull request merged**